blinker/base.py

```diff
@@ -417,7 +417,9 @@ def __repr__(self):
 class Namespace(dict):
     """A mapping of signal names to signals."""
 
-    def signal(self, name, doc=None):
+    signal_cls = NamedSignal
```

Comment:
This is useful and harmless, and I'd readily accept a patch that includes this class member (with a docstring) and a change in `signal` to use it in construction.

Anything fancier (e.g. setting on `Namespace` construction, passing along `signal_cls`constructor args from the `signal()` method, etc.) is probably better handled in user land via a `Namespace` subclass. And given that, I don't see a need to export the global `namespace` instance.